project/src/main/scala/org/broadinstitute/clio/integrationtest/ClioIntegrationTestRunner.scala

```diff
@@ -76,6 +78,35 @@ class ClioIntegrationTestRunner(testClassesDirectory: File,
     *
     * @return The exit code of the integration test.
     */
+  private def tryDockerComposeJenkinsTest(): Try[Int] = Try {
+    runCommandAsync(
```

Comment:
I think we can make this work with a single docker-compose call:
```
$ docker-compose --file docker-compose-jenkins.yml up
```
You'll need to update the compose file to declare that the integration-test container depends on the clio container, so it gets the setup ordering right.